src/OrchardCore/OrchardCore.Abstractions/Json/Dynamic/JsonDynamicValue.cs

```diff
@@ -148,6 +225,41 @@ ulong IConvertible.ToUInt64(IFormatProvider? provider)
         return (ulong)this;
     }
 
+    public static bool operator ==(JsonDynamicValue left, JsonDynamicValue right)
+    {
+        if (ReferenceEquals(left, null))
+        {
+            return ReferenceEquals(right, null);
+        }
+
+        return left.Equals(right);
+    }
+
+    public static bool operator !=(JsonDynamicValue left, JsonDynamicValue right)
+    {
+        return !(left == right);
+    }
```

Comment:
```suggestion
    public static bool operator !=(JsonDynamicValue left, JsonDynamicValue right) => !(left == right);
```

Reply:
I know you prefer expression bodies, but Sebastien removed all of them the last time this file was changed. So I rather keep the format consistent for now.

Reply:
I confirm. When I tried to open the classes and read what they were doing it was so condensed I couldn't understand what I was reading.

What I realized is that if the definition + body fits on a line on the screen then it's fine, if the implementation has to go on the second line then there is no point and it's more readable to just use a standard body.

Reply:
But please don't change anything because "that one char can be removed and it will fit in a single line", there is no readability advantage IMO by squashing everything in a line.